packages/rich-text/src/component/index.native.js

```diff
@@ -765,6 +767,8 @@ export class RichText extends Component {
 						onFocus: () => {},
 					} ) }
 				<RCTAztecView
+					accessibilityLabel={ accessibilityLabel }
+					accessibilityHint={ accessibilityHint }
```

Comment:
Some thoughts about this here: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2368#discussion_r439370912

Reply:
Should we remove the hint completely?

Reply:
I think so 🤔 
The hint doesn't seem to add much when the user is already editing the TextView. Let's keep the hint on the Wrapper view and remove it from Aztec.

Does it sound good?

Reply:
Yeah, sounds like a good idea.